src/pages/dev/send-tokens/interchain-tokens/create-token.mdx

```diff
@@ -0,0 +1,93 @@
+# Create a new Interchain Token
+
+Interchain Tokens are ERC-20 tokens that are available on multiple blockchains. They are created using the [Interchain Token Service](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/InterchainTokenService.sol) and can be used to transfer value between blockchains.
+
+The Interchain Token Service on testnet is deployed to `0xB5FB4BE02232B1bBA4dC8f81dc24C26980dE9e3C` while the Interchain Token Factory is deployed to `0x83a93500d23Fbc3e82B410aD07A6a9F7A0670D66`. You can find the list of the deployed contract addresses to all the networks supported by Axelar [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/testnet.json).
+
+Let's explore two methods to create new Interchain Token(s).
+
+## Create a new Interchain Token(s) using the Interchain Token Portal
+
+The simplest type of Interchain Token is to create a brand new [Interchain Token](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/InterchainTokenService.sol) available on multiple chains. For a detailed guide on how to create an Interchain Token using the Interchain Token Portal, refer to the four-step tutorial provided [here](https://axelar.network/blog/how-to-create-an-interchain-token-with-axelar-in-4-steps).
+
+- Visit the [Interchain Portal](https://testnet.interchain.axelar.dev)
+- Connect your wallet to the portal
+- Select a source network where you have funds available
+- Choose the option to deploy a new Interchain token
+- Add the required details for your new token
+    - Name
+    - Symbol
+    - Decimals
+    - Amount to mint
+- You can also click the advanced option where you can add an account as `Token Minter` and `Salt` value; otherwise, it gets prefilled with the deployer(connected account) address and a random, uniquely generated salt value
+- Select additional chains for your token's availability and optionally add the token amount to mint on each selected chain.
+
+**Congratulations!** Your Interchain Token is now accessible on multiple blockchain networks. This allows for seamless interaction with your token on any of these networks, utilizing familiar methods like send, transfer, and approve, just as with any standard ERC-20 token.
+
+Furthermore, your token is categorized as a [Interchain Token](https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/IInterchainTokenStandard.sol). This classification includes the `interchainTransfer` method, allowing you to transfer it between blockchains.
```

Comment:
I would prefer something like "Furthermore, your token follows the [Interchain Token Standard]. This means that users can call the `interchainTransfer` method on the token itself to transfer between blockchains.